draft-ietf-quic-http.md

```diff
@@ -799,25 +799,34 @@ indicated request.  This trades off network usage against a potential latency
 gain.  HTTP/3 server push is similar to what is described in HTTP/2 {{?HTTP2}},
 but uses different mechanisms.
 
-Each server push is identified by a unique Push ID. This Push ID is used in one
-or more PUSH_PROMISE frames (see {{frame-push-promise}}) that carry the request
-fields, then included with the push stream which ultimately fulfills those
-promises. When the same Push ID is promised on multiple request streams, the
-decompressed request field sections MUST contain the same fields in the
-same order, and both the name and the value in each field MUST be exact
-matches.
-
-Server push is only enabled on a connection when a client sends a MAX_PUSH_ID
-frame; see {{frame-max-push-id}}. A server cannot use server push until it
-receives a MAX_PUSH_ID frame. A client sends additional MAX_PUSH_ID frames to
-control the number of pushes that a server can promise. A server SHOULD use Push
-IDs sequentially, starting at 0. A client MUST treat receipt of a push stream
-with a Push ID that is greater than the maximum Push ID as a connection error of
-type H3_ID_ERROR.
+Each server push is assigned a unique Push ID by the server which is used to
+refer to the push in various contexts throughout the lifetime of the connection.
+
+The Push ID space begins at zero, and ends at a maximum value set by the
+MAX_PUSH_ID frame; see {{frame-max-push-id}}.  In particular, a server is not
+able to push until after the client sends a MAX_PUSH_ID frame.  A client sends
+additional MAX_PUSH_ID frames to control the number of pushes that a server can
+promise.  A server SHOULD use Push IDs sequentially, beginning from zero.  A
```

Comment:
"A client sends additional MAX_PUSH_ID frames...".  This sentence kind of implies that the very first MAX_PUSH_ID frame does not control the number of pushed that a server is allowed to promise, only the subsequent MAX_PUSH_ID frames.  Maybe remove this sentence altogether, because {{frame-max-push-id}} is already doing a great job explaining how the client can send such frames to allow for new push promises as the server fulfills or cancels existing ones?

Reply:
Thank you, https://github.com/quicwg/base-drafts/commit/a248e98f13246c11e6387f62f63c815ce53297e1 solves this nicely.